test/gtest/common/test.cc

```diff
@@ -143,7 +143,7 @@ test_base::count_warns_logger(const char *file, unsigned line, const char *funct
 std::string test_base::format_message(const char *message, va_list ap)
 {
     const size_t buffer_size = ucs_log_get_buffer_size();
-    std::string buf(buffer_size, 'x');
+    std::string buf(buffer_size, '\0');
```

Comment:
sorry, one more thing is missing: need to do buf.resize(strlen(buf.c_str())

Reply:
instead of just vector or auto_ptr :)
ok, fixed